src/families/cosmos/libcore-postBuildAccount.js

```diff
@@ -11,6 +11,39 @@ import {
 import { promiseAllBatched } from "../../promise";
 import { getMaxEstimatedBalance } from "./logic";
 
+const addCoreOperationToAccountOperations = async (
+  accountOpsFeeStore: Promise<{
+    ops: { [tx_hash: string]: [Operation] },
+    fees: { [tx_hash: string]: string },
+  }>,
+  newCoreOp: Operation
+): Promise<{
+  ops: { [id: string]: [Operation] },
+  fees: { [tx_hash: string]: BigNumber },
+}> => {
+  let result = await accountOpsFeeStore;
+  const id = newCoreOp.id;
+  // This split comes from the code in src/families/cosmos/libcore-buildOperation.js
+  const txHash = id.split("-")[1];
```

Comment:
I think you can get the hash from the operation no ?

Reply:
`newCoreOp.hash` should work I think

Reply:
Didn't think of that <_< but yeah I really just want the txHash